uint8 id
---
uint16 realtimeTick
int32 result
